React/Views/ScrollView/RCTScrollContentView.m

```diff
@@ -54,6 +54,15 @@ - (void)reactSetFrame:(CGRect)frame
            horizontalScrollerHeight:verticalScrollerWidth];
     [[[scrollView bridge] uiManager] setLocalData:localData forView:self];
   }
+    
+  NSMutableArray *subViews = [[NSMutableArray alloc] init];
+  for (NSView *view in [self subviews]) {
```

Comment:
are there actually subviews that are not rctviews that we need to do these filters?

Reply:
![image](https://user-images.githubusercontent.com/67026167/183759368-d71a9ae1-df2a-44dd-b19c-f3558b54acc6.png)
Every row in the subviews gets backed by a RCTView so I think it should be safe?

Reply:
then we can just pass in the subview for children without htis copy of an array?

Reply:
We filter out unwanted subviews like NSScroller and populate the copy array with subviews that are list items only. Does that make sense?